coalib/bearlib/spacing/SpacingHelper.py

```diff
@@ -86,11 +86,28 @@ def replace_spaces_with_tabs(self, line: str):
         :param line: The string with spaces to replace.
         :return:     The converted string.
         """
+        def previous_whitespace():
+            # Find the previous real character, and remaining chars to fill tab
```

Comment:
why nesting functions :O what about putting this into class-level and prepending an underscore to the function name?